books/sociology/bake

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '7.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :sociology) do |doc|
+  include Kitchen::Directions
+
+  # Set overrides
+  doc.selectors.override(
+    page_summary: 'section.section-summary',
+    reference: 'section.references'
+  )
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  # Remove data-type="description" from body metadata placed in Sociology 2e
+  book.body.search('div[data-type="metadata"] > div[data-type="description"]').trash
+
+  BakePreface.v1(book: book)
+
+  # Bake NumberedTable in Preface
+  book.pages('$.preface').tables('$:not(.unnumbered)').each do |table|
+    BakeNumberedTable.v1(table: table, number: "#{table.count_in(:page)}")
+  end
+
+  BakeChapterTitle.v1(book: book)
+  BakeChapterIntroductions.v1(book: book)
+
+  # Bake EoC sections
+  book.chapters.each do |chapter|
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata, klass: 'section-summary')
+    MoveExercisesToEOC.v3(chapter: chapter, metadata_source: metadata, klass: 'section-quiz')
+    MoveExercisesToEOC.v3(chapter: chapter, metadata_source: metadata, klass: 'short-answer')
+    BakeFurtherResearch.v1(chapter: chapter, metadata_source: metadata)
+    chapter.composite_pages.each do |composite_page|
+      composite_page.search('section.short-answer, section.section-quiz').exercises.each do |exercise|
+        BakeNumberedExercise.v1(exercise: exercise,
+                                number: exercise.count_in(:composite_page),
+                                suppress_solution_if: :even?,
+                                note_suppressed_solutions: true)
+      end
+    end
+    BakeChapterReferences.v1(chapter: chapter, metadata_source: metadata)
+
+  end
+
+  # Bake Answer Key
+  if book.contains?(' div[data-type="solution"]') # Used only in Sociology 3e, Sociology 2e has supressed (commented out) all solutions
```

Comment:
is the space here at the start of the selector intentional?

This could be expensive as it is searching the entire book.  Could have

```ruby
if 3 == edition
  # do this stuff, no comment needed b/c the "if" clause says it for you
end
```

Reply:
No it wasn't intentional. Thank you!